aws-terraform/README.md

```diff
@@ -23,16 +23,25 @@ Init mode will guide the administrator through an interactive process which gene
 
 ## OP-SCIM Bridge 
 
-The 1Password SCIM Bridge is distributed as the Debian package and can be installed during the deployment process using the following repo configuration: `deb https://apt.agilebits.com/op-scim/ stable op-scim`. [Repo GPG key](apt.agilebits.com_repo_key.asc).  
+The 1Password SCIM Bridge is distributed as the Debian package and can be installed during the deployment process using the following repo configuration: `deb https://apt.agilebits.com/op-scim/ stable op-scim`. [Repo GPG key](https://apt.agilebits.com/gpg.key) can be downloaded from the same repo.  
```

Comment:
s/repo/repository/

Reply:
also,
better to show it on a separate line/block

```
deb https://apt.agilebits.com/op-scim/ stable op-scim
```

or even instruction on how and where put that line:

```
echo "deb https://apt.agilebits.com/op-scim/ stable op-scim" > /etc/apt/sources.list.d/op-scim.list
```

Plus instruction how to add repo(sitrory) key

```
curl -s https://apt.agilebits.com/gpg.key | apt-key add -
```